docs/getting_started/tutorial.rst

```diff
@@ -202,21 +252,85 @@ Create a template at ``blog/templates/blog/blog_page.html``:
         <div class="intro">{{ page.intro }}</div>
 
         {{ page.body|richtext }}
+
+        <p><a href="{{ page.get_parent.url }}">Return to blog</a></p>
+
     {% endblock %}
 
-Run ``python manage.py makemigrations`` and ``python manage.py migrate``.
+Note the use of Wagtail's built-in ``get_parent()`` method to obtain the
+URL of the blog this post is a part of.
+
+Now create a few blog posts as children of ``BlogIndexPage.``
+Be sure to select type "BlogPage" when creating your posts.
 
-.. figure:: ../_static/images/tutorial/tutorial_4.png
-   :alt: Create page screen
+.. figure:: ../_static/images/tutorial/tutorial_4a.png
+   :alt: Create blog post as child of BlogIndex
+
+.. figure:: ../_static/images/tutorial/tutorial_4b.png
+  :alt: Choose type BlogPost
+
```

Comment:
The screenshots here incorrectly show the blog post being created as a child of Homepage.